guidelines/REUSE.md

```diff
@@ -1,85 +1,100 @@
 # Reuse Philosophy
+
 We encourage reuse and patterns through references.
 
 ## What is reusable
+
 The following types are reusable, as defined by the spec:
 
-* Operations
 * Parameters
-* Responses
 * Models (or Schema Objects in general)
+* Responses
+* Operations (_Operations can only be remote references_)
 
 ## Reuse strategy
-When reusing components in an API design, a pointer is created from the definition to target design.  The references are maintained in the structure, and can be updated by modifying the source definitions.  This is different from a "copy on design" approach where references are injected into the design itself.
 
-The reuse technique is transparent between JSON or YAML and is lossless when converting between the two.
+When reusing components in an API design, a reference is created from the definition to target entity. The references are maintained in the structure, and can be updated by modifying the source definitions. This is different from a "copy on design" approach where references are injected into the design itself.
```

Comment:
I personally find this paragraph to be painful to read.  Since most of the verbiage predates this PR, this is not necessarily a complaint about the PR but instead a suggestion in general.  Here are my complaints:

* The terms _"component"_ and _"entity"_ are not used anywhere else in specification documentation so they are confusing here.  I know what is being said because I know Swagger very well but someone potentially new to Swagger might not get it.
* _"The references are maintained in the structure"_ seems _duh_ to me because where else would they be maintained?
* _"API design"_ in the first sentence should probably be _"API design document"_
* The last sentence really isn't helpful because that's an implementation detail, and some implementations may very well inject the dereferenced value into the document
* Since OpenAPI references work identically regardless of JSON or YAML, I don't think we need to mention it
* YAML anchors injecting the dereferenced value into its resulting object is an implementation detail.  To the end user, they would get the same result using YAML anchors as they would with JSON References.  While I do realize that the resulting in-memory representation can be different when using JSON References vs. YAML anchors, I don't know if this is worth mentioning since it's a tooling thing that most API design document authors will never see.

I wouldn't be very useful if I just critiqued so here is a version that addresses these concerns.  I'm not saying we need to use this version but I am hoping that it will be useful in coming to some sort of agreement:

As you author your API design documents, there may come a time when you have common object definitions that you would like to reuse to avoid duplication.  *(For example, imagine you have multiple path definitions that each share a common path parameter, or you have a common response structure.)*  The OpenAPI specification allows you to reuse common object definitions through the use of _"references"_.

A reference is technically just a placeholder, a construct in your API design document that says _"the real content for this portion of the document is defined elsewhere"_.  To create a reference, at the location in your document where you want to reuse some other definition, you create an object that has a `$ref` property whose value is a `URI` pointing to where the definition is *(more on this in later sections)*.  Here are two examples:

**JSON** *(JavaScript syntax used to allow comments for demonstration purposes)*
```js
{
  // ...
  definitions: {
    Person: {
      type: 'object',
      properties: {
        friends: {
          type: 'array',
          items: {
            $ref: '#/definitions/Person'
          }
        }
      }
    }
  }
}
```

**YAML**
```yaml
# ...
definitions:
  Person:
    type: object
    properties:
      friends:
        type: array
        items:
          $ref: '#/definitions/Person'
```

In reality, the OpenAPI _"reference"_ concept is not something we came up with.  In fact, it is known as a [JSON Reference](http://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03).  The major difference between JSON References and OpenAPI references is that JSON References can point to anything, anywhere but OpenAPI references are intended only to point to definitions as defined in the section above.

**Note:** Those of you familiar with YAML know that YAML already has a feature very similar to what is described above.  YAML _"anchors"_ also allow you to reuse common definitions/values but since that is a YAML feature, we will not go into further detail on the matter.